docker/Dockerfile

```diff
@@ -37,6 +37,33 @@ RUN conda config --set show_channel_urls true && \
     conda info --all && \
     conda list
 
+# install tools for using OpenStreetMap
+RUN apt-get update && apt-get install -y osm2pgsql osmctools
```

Comment:
Need to put all the RUN commands before line 68 into a single RUN command (see lines 31-38 as example) to prevent image bloat.

Reply:
This has also been done -- with exception of the calls to python and ipython (as per main comment below; I think these were in original docker file, so assume intended to be separate).